test/golang-tests.js

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2018, Patrick Quist
```

Comment:
I'm starting to use "201x, Compiler Explorer Authors" in case you want to do the same, although it's not required :)